### PR TITLE
Add plugin boilerplate to indicate the plugin owner

### PR DESCRIPTION
**Pull request by hughsie**
A frequent anxiety heard from new vendors is that anybody can edit any plugin as the code is "open source". To allay that fear add an 'Owners' section in each plugin README.md to specify people that should be consulted for specific reviews or compliance or functional testing.

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [ ] Feature
- [X] Documentation

### CODE REVIEW

**Comment by hughsie** [comment]
@superm1 if you agree with the wording and concept I'll do all the grunt work for the other plugins too.

File: plugins/amd-gpu/README.md [hunk 1]
```diff
@@ -63,3 +63,10 @@ interfaces with the hardware.
       PSP ~~~ kernel
       kernel ~~~ fwupdengine
 ```
+
+## Owners
+
+Any anonymous user can submit a pull request to modify this plugin, but the following people should
```
Comment (superm1) [inline plugins/amd-gpu/README.md:69]: It's not "really" anonymous - they're authenticated GitHub users.

How about just "anyone"?

**Review by superm1 — COMMENTED** [review]
I'm fine with it

File: plugins/amd-gpu/README.md (hunk 1 above)
Comment (hughsie) [reply plugins/amd-gpu/README.md:69]: Makes sense -- fixed -- thanks.

**Comment by hughsie** [comment]
@superm1 well, that's an hour of my life I'm never going to get back! :)

File: plugins/dell-dock/README.md [hunk 2]
```diff
@@ -192,3 +192,10 @@ This plugin has been available since fwupd version `1.2.0`.
       I2C~~~USB1
       I2C~~~USB2
 ```
+
+## Owners
+
+Anyone can submit a pull request to modify this plugin, but the following people should be
+consulted before making major or functional changes:
+
+* Mario Limonciello: @superm1
```
Comment (superm1) [inline plugins/dell-dock/README.md:201]: Probably should add crag here, he's more in the know than me these days

**Review by superm1 — APPROVED** [review]

File: plugins/dell-dock/README.md (hunk 2 above)
Comment (hughsie) [reply plugins/dell-dock/README.md:201]: prefixed, thanks.

**Pull request merged**